components/dash-core-components/package.json

```diff
@@ -42,12 +42,13 @@
     "@fortawesome/react-fontawesome": "^0.1.16",
     "base64-js": "^1.5.1",
     "color": "^4.1.0",
+    "d3-format": "^1.4.5",
```

Comment:
For now I used the same version as loaded by `plotly.js`. 
Using newer versions of this module could also be considered in future.